packages/expo-cli/src/credentials/provider/android.ts

```diff
@@ -0,0 +1,107 @@
+import { SetupAndroidKeystore } from '../views/SetupAndroidKeystore';
+import { Keystore } from '../credentials';
+import { runCredentialsManager } from '../route';
+import { Context } from '../context';
+import { credentialsJson } from '../local';
+import { CredentialsProvider } from './provider';
+
+export interface AndroidCredentials {
+  keystore: Keystore;
+}
+
+interface Options {
+  projectName: string;
+  accountName: string;
+}
+
+export class AndroidCredentialsProvider implements CredentialsProvider {
+  public readonly platform = 'android';
+  private readonly ctx = new Context();
+  private credentials?: AndroidCredentials;
+
+  constructor(private projectDir: string, private options: Options) {}
+
+  get projectFullName(): string {
+    const { projectName, accountName } = this.options;
+    return `@${accountName}/${projectName}`;
+  }
```

Comment:
This function is the same as in the iOS provider.

Reply:
Yes, I would prefer to keep CredentialsProvider as an interface, not a class. I think that it's small enough helper function that it can be duplicated. normally I would write it inline, but access `this.options.someValue` make it a quite long statement